File: src/main/java/com/matchFit/post/dto/UpdatePostRequestDto.java
package com.matchFit.post.dto;

import com.matchFit.post.entity.Sports;
import com.matchFit.post.entity.Town;
import com.matchFit.user.entity.Gender;
import java.time.LocalDateTime;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UpdatePostRequestDto {
    private String title;
    private String description;
    private String location;
    private LocalDateTime date;
    private Integer maxPeople;
    private Gender gender;
    private Integer cost;
    private String imageUrl;
    private Sports sports;
    private Town town;
    private Boolean removeImage;
}

File: src/main/kotlin/com/matchFit/post/dto/UpdatePostRequestDto.kt
package com.matchFit.post.dto

import com.matchFit.post.entity.Sports
import com.matchFit.post.entity.Town
import com.matchFit.user.entity.Gender
import java.time.LocalDateTime

class UpdatePostRequestDto {
    var title: String? = null
    var description: String? = null
    var location: String? = null
    var date: LocalDateTime? = null
    var maxPeople: Int? = null
    var gender: Gender? = null
    var cost: Int? = null
    var imageUrl: String? = null
    var sports: Sports? = null
    var town: Town? = null
    var removeImage: Boolean? = null
}
